doc/release/0.2.rst

```diff
@@ -8,7 +8,9 @@ Release Date: TBD
 
 What's new in ARMI
 ------------------
-#. TBD
+#. TBD: Composites can now be nested inside other Composites
+#. TBD: The global CaseSettings object has been removed
+#. The method ``Material.density3`` is now called ``density``, and the old ``density`` is now called ``pseudoDensity``. (`PR#1163 <https://github.com/terrapower/armi/pull/1163>`_)
```

Comment:
These changes are unrelated to this PR? Do you need to merge main?

Reply:
Unrelated to this PR. Sure, I can move them to their own PR.

I just want to release ARMI version `0.3.0` as soon as these three topics are done (though that third one is probably more like "Materials were reworked to be more consistent").

So I will want to put those three TBDs at the top of our release notes as part of our great ARMI release planning.

So... sure, I can remove it.  But expect to see it again soon!

Reply:
Removed!